rdkit/Chem/GraphDescriptors.py

```diff
@@ -34,28 +33,22 @@ def _VertexDegrees(mat, onlyOnes=0):
 
   """
   if not onlyOnes:
-    res = sum(mat)
-  else:
-    res = sum(numpy.equal(mat, 1))
-  return res
+    return sum(mat)
+  return sum(numpy.equal(mat, 1))
 
 
 def _NumAdjacencies(mol, dMat):
   """  *Internal Use Only*
 
   """
-  res = mol.GetNumBonds()
-  return res
+  return mol.GetNumBonds()
 
 
 def _GetCountDict(arr):
   """  *Internal Use Only*
 
   """
-  res = {}
-  for v in arr:
-    res[v] = res.get(v, 0) + 1
-  return res
+  return dict(Counter(arr))
```

Comment:
I don't think this will be any faster.

Reply:
OK, I got your idea! I would fix it in the next patch. Thank you for your review.

Reply:
[FYI]: This function is not called from anywhere, so I just convert it to `Counter(arr)`. The reason I still keep this change is also help that adding third-party library is safe to do so at any other modules apart from `Descriptors.py`.